Src/LiquidProjections.NHibernate/NHibernateEventMapConfigurator.cs

```diff
@@ -74,6 +81,11 @@ internal sealed class NHibernateEventMapConfigurator<TProjection, TKey>
             }
             else
             {
+                if (!context.Session.Contains(projection))

```

Comment:
What if somehow the projector has loaded the projection from the database without trying to get if from the cache first? In that case the session contains a different instance of the same projection.

Reply:
That would mean somebody tinkered with the session directly without us knowing about it. How do you suggest we handle that? Update the cache with the existing item? Or overwrite the session regardless?